src/ontology/cl-edit.owl

```diff
@@ -2078,6 +2078,7 @@ Declaration(Class(obo:CL_0019028))
 Declaration(Class(obo:CL_0019029))
 Declaration(Class(obo:CL_0019031))
 Declaration(Class(obo:CL_0019032))
+Declaration(Class(obo:CL_0300023))
```

Comment:
Is there a reason why this new class is in this id range?

Reply:
Good catch. It may be due to working in another ontology and I did not update the range when in CL or a typo. Will update...